frontend/app/brains-management/[brainId]/components/BrainManagementTabs/components/SettingsTab/components/PublicPrompts.tsx

```diff
@@ -31,27 +33,81 @@ export const PublicPrompts = ({
     }
   };
 
+  return (
+    <CustomSelect
+      options={publicPrompts}
+      onChange={handleChange}
+      onSelect={onSelect}
+    />
+  );
+};
+
+type CustomSelectProps = {
```

Comment:
Let's split this file. Let's have a Prompt folder with components, hooks and types in different files